public/coffee/ide/editor/directives/aceEditor/auto-complete/SuggestionManager.coffee

```diff
@@ -39,15 +39,18 @@ define [], () ->
 			return commands
 
 		# Ignore single letter commands since auto complete is moot then.
-		commandRegex: /\\([a-zA-Z][a-zA-Z]+)/
+		# Ignore commands which occur after a comment character '%'
+		commandRegex: /^([^%]*?)\\([a-zA-Z][a-zA-Z]+)/m
```

Comment:
I also worry this regexp could be expensive now since it has to backtrack? How about blanking all comments first, then running the current regex? That might be easier... not sure.